lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -463,9 +463,9 @@ private function parseColumnCollationFromSQL($column, $sql)
     private function parseColumnCommentFromSQL($column, $sql)
     {
         if (preg_match(
-            '{[\s(,](?:'.preg_quote($this->_platform->quoteSingleIdentifier($column)).'|'.preg_quote($column).')
+            '/[\s(,](?:'.$this->pregQuote($this->_platform->quoteSingleIdentifier($column)).'|'.$this->pregQuote($column).')
```

Comment:
This is what I told you about CS violations: https://travis-ci.org/doctrine/dbal/jobs/285994030#L2412 can you please fix it?